.github/workflows/resources/poseidon-minimal.service

```diff
@@ -8,5 +8,11 @@ Requires=poseidon.socket
 [Service]
 WorkingDirectory=${GITHUB_WORKSPACE}
 ExecStart=${GITHUB_WORKSPACE}/poseidon
-Restart=always
 Environment="POSEIDON_SERVER_SYSTEMDSOCKETACTIVATION=TRUE"
+Environment="GOCOVERDIR=${GITHUB_WORKSPACE}/${GOCOVERDIR}"
```

Comment:
What do you think about extracting this variable `GOCOVERDIR` and the other ones `.github/workflows/ci.yml` (lines 126-132) to a dedicated file? 

Then, we could use this file in the systemd service with `EnvironmentFile=<Path>` and in the remaining parts with one of the solutions sketched [here](https://stackoverflow.com/questions/19331497/set-environment-variables-from-file-of-key-value-pairs)?

My main goal would be to reduce duplication of the variables. Otherwise, I would feat that we miss adjusting the variables again in the future.